internal/admission/validator.go

```diff
@@ -281,9 +281,6 @@ func (validator KongHTTPValidator) ValidatePlugin(
 	ctx context.Context,
 	k8sPlugin kongv1.KongPlugin,
 ) (bool, string, error) {
-	if k8sPlugin.PluginName == "" {
```

Comment:
I have a doubt that we could not remove the validation in webhooks here. The CEL is enabled in k8s 1.23 and enabled by default in 1.25, so if we still need to support 1.22+, we could not remove the validation in webhooks. WDYT? @pmalek

Reply:
According to https://docs.konghq.com/kubernetes-ingress-controller/latest/reference/version-compatibility/#general and a [long chat on slack](https://kongstrong.slack.com/archives/C011RQPHDC7/p1698757312230499) we're supporting 1.25+ with v3.0

Reply:
yes. we're supporting 1.25+ with v3.0

Reply:
Given that we already have this field set as required I'm OK with this change 👍 https://github.com/Kong/kubernetes-ingress-controller/blob/314951c3279dcd38cb7de7e53f71969379a0340a/pkg/apis/configuration/v1/kongplugin_types.go#L67